lib/grizzly/node.ex

```diff
@@ -15,8 +15,8 @@ defmodule Grizzly.Node do
 
   The response to this command is the `NodeInfoCacheReport` command
   """
-  @spec get_node_info(ZWave.node_id()) :: Grizzly.send_command_response()
-  def get_node_info(node_id) do
+  @spec get_info(ZWave.node_id()) :: Grizzly.send_command_response()
+  def get_info(node_id) do
```

Comment:
```suggestion
  @deprecated "Use Grizzly.Node.get_info/1 instead"
  def get_node_info(node_id), do: get_info(node_id)

  @spec get_info(ZWave.node_id()) :: Grizzly.send_command_response()
  def get_info(node_id) do
```
Maybe just do a deprecation here if worried about a breaking change? Gives a bit of time to adjust